lib/ably/AblyRest.php

```diff
@@ -124,7 +125,8 @@ public function authorise( $options = array(), $params = array(), $force = false
             if ( $this->token->expires > $this->timestamp() ) {
                 if ( !$force ) {
                     # using cached token
-                    $this->log_action( 'authorise()', sprintf("\tusing cached token; expires = %s\n\tfor humans token expires on %s", $this->token->expires, gmdate("r",$this->token->expires)) );
+                    $this->log_action('authorise()', sprintf("\tusing cached token; expires = %s\n\tfor humans token expires on %s",
```

Comment:
Is there no log level?  This type of log entry is typically debug level in other client libs.  Out of interest, how does it work in PHP?